editor/src/missing-types/index.d.ts

```diff
@@ -16,7 +16,7 @@ declare module '@babel/plugin-transform-modules-commonjs'
 declare module '@babel/plugin-proposal-export-namespace-from'
 
 declare module 'lodash.clamp' {
-  export const clamp = (number: number, lower: number, upper: number) => number
+  export default clamp = (number: number, lower: number, upper: number) => number
   export const clamp = (number: number, upper: number) => number
```

Comment:
Should this definition change as well?

Reply:
I've checked the lodash library and updated this